events/activemq.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+package events
+
+type ActiveMQEvent struct {
+	EventSource    string            `json:"eventSource"`
+	EventSourceARN string            `json:"eventSourceArn"`
+	Messages       []ActiveMQMessage `json:"messages"`
+}
+
+type ActiveMQMessage struct {
+	MessageID     string              `json:"messageID"`
+	MessageType   string              `json:"messageType"`
+	Timestamp     int64               `json:"timestamp"`
+	DeliveryMode  int                 `json:"deliveryMode"`
+	CorrelationID string              `json:"correlationID"`
+	ReplyTo       string              `json:"replyTo"`
+	Destination   ActiveMQDestination `json:"destination"`
+	Redelivered   bool                `json:"redelivered"`
+	Type          string              `json:"type"`
+	Expiration    int64               `json:"expiration"`
+	Priority      int                 `json:"priority"`
+	Data          string              `json:"data"`
```

Comment:
@shask-amazon from the docs and sample data, this seems to always be base64 encoded? What do you think about changing the type to `[]byte` so that the encode/decode is automatic for users?

Reply:
Fair point. I chose string to be consistent with POJO -  https://tinyurl.com/y8wde5pn